backends/tfhe-cuda-backend/cuda/include/integer.h

```diff
@@ -1535,8 +1535,8 @@ template <typename Torus> struct int_arithmetic_scalar_shift_buffer {
 
   Torus *tmp_rotated;
 
-  cudaStream_t local_stream_1;
-  cudaStream_t local_stream_2;
+  cudaStream_t *local_streams_1;
+  cudaStream_t *local_streams_2;
```

Comment:
Shouldn't we use vectors for consistency with `int_radix_lut`?

Reply:
Yes, I'll change it :+1:

Reply:
Ok so it's not so easy to change: we pass streams with `cudaStream_t *` type that cross the c++/Rust interface, so it's much easier to keep it this way.